resources/docker-compose/apim/docker-compose.yaml

```diff
@@ -70,7 +70,6 @@ services:
     volumes:
       - ../resources/enforcer/security:/home/wso2/security
       - ./conf/log4j2.properties:/home/wso2/conf/log4j2.properties
-      - ../resources/enforcer/dropins:/home/wso2/lib/dropins
```

Comment:
I think we can keep this mount in the docker-compose setup, so it is easy for testing purposes.
For eg: testing a custom filter, we have to just add the jar to `docker-compose/resources/enforcer/dropins`
Removing the mount from k8s config looks fine for me. @VirajSalaka wdyt?

Reply:
Event tests are failed due to the same reason
```
Error:  Failures: 
Error:    AllHttpMethodsForWildcardTestCase.testTrailingSlashHEAD:145 » NullPointer
Error:    APIPolicyTestCase.testCustomOPAAPIPolicyFailedValidation:245 Response code mismatched expected [403] but found [500]
Error:    APIPolicyTestCase.testCustomOPAAPIPolicySuccessValidation:224 Response code mismatched expected [200] but found [500]
Error:    CustomFilterTestCase.testAddHeaderViaFilter:60 » JSON JSONObject["Custom-heade...
Error:    CustomFilterTestCase.testDynamicEndpointWithinFilter:146 Response body mismatched for dynamic endpoint expected [{"id":111111155, "category":{ "id":0, "name":"ABCD" }, "name":"doggieUpdated", "photoUrls":[ "SampleImage1.png" ], "tags":[ { "id":0, "name":"TestTag1" } ], "status":"pending"}] but found [{"id":111111148, "category":{ "id":0, "name":"ABCD" }, "name":"doggieUpdated", "photoUrls":[ "SampleImage1.png" ], "tags":[ { "id":0, "name":"TestTag1" } ], "status":"pending"}]
Error:    CustomFilterTestCase.testFilterConfigProperties:127 » JSON JSONObject["Fookey"...
Error:    CustomFilterTestCase.testPathParameterReadWithinFilter:77 » JSON JSONObject["H...
Error:    CustomFilterTestCase.testQueryParameterReadWithinFilter:94 » JSON JSONObject["...
Error:    CustomFilterTestCase.testRequestBodyPass:161 Header is not attached from the custom filter. expected [true] but found [false]
Error:    CustomJwtTransformerTestCase.testCustomJwtClaimMapping:59 » JSON JSONObject["C...
[INFO] 
Error:  Tests run: 384, Failures: 10, Errors: 0, Skipped: 0
```